lib/cdo/shared_constants/pd/shared_workshop_constants.rb

```diff
@@ -228,11 +228,18 @@ module SharedWorkshopConstants
     }.freeze
 
     ACTIVE_COURSE_WORKSHOPS = {
-      CSD: 'CS Discoveries',
-      CSP: 'CS Principles',
-      CSA: 'Computer Science A'
+      CSD: COURSE_CSD,
```

Comment:
This was just a clean up. There are a lot of constants in this area of the code that could use to be cleaned up